argo-data-submission-wf/main.nf

```diff
@@ -40,10 +41,13 @@ params.cleanup = true
 // ArgoDataSubmissionWf
 params.study_id=""
 params.download_mode="local"
-params.song_container = ""
-params.song_container_version = ""
-params.score_container = ""
-params.score_container_version = ""
+params.song_container = "ghcr.io/overture-stack/song-client"
+params.song_container_version = "latest"
+params.score_container = "ghcr.io/overture-stack/score"
+params.score_container_version = "latest"
```

Comment:
Better use specific versions for song/score instead of `latest` in workflow, as `latest` is kind of like a moving tag.